core-bundle/src/Event/DbafsMetadataEvent.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Event;
+
+class DbafsMetadataEvent
```

Comment:
Note to myself: there should probably be two events: one for normalization, one for denormalization. (Maybe use serializer component?)

Reply:
implemented in e21aa8064f352740870ee50415f3d3d37121d29a